config/config.go

```diff
@@ -485,6 +488,7 @@ func DefaultConsensusConfig() *ConsensusConfig {
 		CreateEmptyBlocksInterval:   0,
 		PeerGossipSleepDuration:     100,
 		PeerQueryMaj23SleepDuration: 2000,
+		BlockTimeIota:               1000,
```

Comment:
I think this needs to be a consensus parameter since it determines validity for blocks